src/Symfony/Bundle/FrameworkBundle/Test/HttpClientAssertionsTrait.php

```diff
@@ -31,61 +31,64 @@ public static function assertHttpClientRequest(string $expectedUrl, string $expe
 
         /** @var HttpClientDataCollector $httpClientDataCollector */
         $httpClientDataCollector = $profile->getCollector('http_client');
-        $expectedRequestHasBeenFound = false;
-
+        
+        // Check if the specified HttpClient exists
         if (!\array_key_exists($httpClientId, $httpClientDataCollector->getClients())) {
-            static::fail(\sprintf('HttpClient "%s" is not registered.', $httpClientId));
+            static::fail(sprintf('HttpClient "%s" is not registered.', $httpClientId));
```

Comment:
please revert all CS changes, they make reviewing the PR difficult and are unrelated to the proposed change